Interni/StudioDiFattibilità/informazioni.tex

```diff
@@ -0,0 +1,21 @@
+\newcommand{\Titolo}{Studio di fattibilità}
+
+\newcommand{\Gruppo}{Mercury Seven}
+
+\newcommand{\VersioneDocumento}{\versNdP}
+
+\newcommand{\ACapoRedazione}{\textit{Albiero Davide}\newline \textit{Calgaro Giosuè}}
+
+\newcommand{\Verifica}{Daniele Giachetto}
+
+\newcommand{\Responsabile}{Approvatore}
+
+\newcommand{\Distribuzione}{\textit{Prof. Tullio Vardanega}\newline \textit{Prof. Riccardo Cardin}\newline Gruppo \Gruppo{}}
+
+\newcommand{\Uso}{Interno}
+
+\newcommand{\NomeProgetto}{SSD: Soluzioni di sincronizzazione desktop}
```

Comment:
Il nome del progetto non risulta gradevole, sarebbe da spostare in una riga per lui.

Reply:
@Dema00 riesci a risolvere?